src/components/clusterConfiguration/BasicNetworkFields.tsx

```diff
@@ -42,6 +59,26 @@ const getVipHelperSuffix = (
   return '';
 };
 
+const getVipValidationsById = (
+  validationsInfoString?: Cluster['validationsInfo'],
+): { [key: string]: string | undefined } => {
+  const validationsInfo = stringToJSON<ValidationsInfo>(validationsInfoString) || {
+    hostsData: [],
+    network: [],
+    configuration: [],
+  };
+  const failedDhcpAllocationMessageStub = 'VIP IP allocation from DHCP server has been timed out';
+  return validationsInfo.network.reduce((lookup, validation) => {
+    if (['api-vip-defined', 'ingress-vip-defined'].includes(validation.id)) {
+      lookup[validation.id] =
+        validation.status === 'failure' && validation.message.match(failedDhcpAllocationMessageStub)
```

Comment:
Having this, the only error we pass is the one matching `failedDhcpAllocationMessageStub`.
What if new errors are added in the future?

I think we should reverse the condition and black-list (read: use a list of skipped) messages which will not be shown in the UI. And report the rest.

Reply:
I tried doing this and it lead to displaying an error message when there was no dhcp allocation failure. In both cases we're matching the string which is error prone. I've decided to handle that specific message as Ori mentioned that the only way the allocation fails is the timeout (IOW the backend always reports timeout regardless of how the dhcp allocation fails because the allocation job timeouts and causes the failure)